external-import/misp/src/misp.py

```diff
@@ -132,6 +132,13 @@ def __init__(self):
             False,
             "timestamp",
         )
+        self.misp_filter_date_attribute = get_config_variable(
+            "MISP_DATE_FILTER_FIELD",
+            ["misp", "date_filter_field"],
+            config,
+            False,
+            "date_from",
```

Comment:
I think it's `from_date` like everywhere?

```suggestion
            "from_date",
```

Reply:
This a typo from my side, in PyMisp it should map to `date_from` 

https://github.com/MISP/PyMISP/blob/9408cf180da9986781c739373aab28fa5fc789ac/pymisp/api.py#L2765